### PR TITLE
feat: Authentication and screens :

### PR DESCRIPTION
**Pull request by Tanjum786**
> This PR contains Authentication and demo screens of

-   Playlist
- Watch  later page
- History page
- liked page

>  Created login signup page 
-  fetch data from DB

### CODE REVIEW

File: src/Components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+export const Footer = () => {
+  return (
+    <div>
+      <footer className="footer">
+        <div className="footer-header dis_flex">
+          <p>Made by Tanjum</p>
+        </div>
+        <ul className="social_media_links dis_flex gap_s">
+          <li className="list-item-inline">
+            <a href="https://github.com/Tanjum786" target="blank_">
```
Comment (shashidwivedi) [inline src/Components/Footer/Footer.jsx]: I think it's a typo here for target="_blank"?

**Review by shashidwivedi — APPROVED** [review]
Everything else looks good. Can be merged

**Review by therajatg — APPROVED** [review]
code looks good

File: src/Components/Footer/Footer.jsx (hunk 1 above)
Comment (Tanjum786) [reply src/Components/Footer/Footer.jsx]: yeah i will correct it

**Pull request merged**